daemon/execdriver/driver.go

```diff
@@ -72,13 +76,13 @@ type Network struct {
 	HostNetworking bool              `json:"host_networking"`
 }
 
-// IPC settings of the container
+// Ipc settings of the container
 type Ipc struct {
 	ContainerID string `json:"container_id"` // id of the container to join ipc.
 	HostIpc     bool   `json:"host_ipc"`
 }
 
-// PID settings of the container
+// Pid settings of the container
```

Comment:
Same, this should mention this is specifically about PID namespace settings, and since this only supports host PID, it's really about specifying if the container should join the host PID namespace.